src/helpers/ASCIIArts.ts

```diff
@@ -0,0 +1,51 @@
+export const goat = `
+             ,--._,--.
+           ,'  ,'   ,-\`.
+(\`-.__    /  ,'   /
+ \`.   \`--'        \\__,--'-.
+   \`--/       ,-.  ______/
+     (o-.     ,o- /
+      \`. ;        \\
+       |:          \\
+      ,'\`       ,   \\
+     (o o ,  --'     :
+      \\--','.        ;
+       \`;;  :       /
+        ;'  ;  ,' ,'
+        ,','  :  '
+        \\ \\   :
+         \`
+      `;
+
+
+export const train =
+  `
+                      (@@) (  ) (@)  ( )  @@    ()    @     O     @     O      @
+                      (   )
+                  (@@@@)
+               (    )
+           (@@@)
+         ====        ________                ___________
+     _D _|  |_______/        \\__I_I_____===__|_________|
+      |(_)---  |   H\\________/ |   |        =|___ ___|      _________________
+      /     |  |   H  |  |     |   |         ||_| |_||     _|                \_____A
+     |      |  |   H  |__--------------------| [___] |   =|                        |
+     | ________|___H__/__|_____/[][]~\\_______|       |   -|                        |
+     |/ |   |-----------I_____I [][] []  D   |=======|____|________________________|_
+   __/ =| o |=-~~\\  /~~\\  /~~\\  /~~\\ ____Y___________|__|__________________________|_
+    |/-=|___|=O=====O=====O=====O   |_____/~\\___/          |_D__D__D_|  |_D__D__D_|
+     \\_/      \\__/  \\__/  \\__/  \\__/      \\_/               \\_/   \\_/    \\_/   \\_/
+      `;
+
+let dogArr = [
```

Comment:
why have the dog in a different format than the other arts? they are all consts and string literals while this one is an array of normal strings that get joined